deploy/template.yaml

```diff
@@ -416,13 +416,15 @@ Resources:
               DATA_TXMA_SQS: "/perfTest/data/txma/perfSQS"
               IDENTITY_ADDRESS_URL: "/perfTest/identity/orange/addressUrl"
               IDENTITY_BUILD_CORE_URL: "/perfTest/identity/build/coreUrl"
+              IDENTITY_BUILD_ORCH_STUB_URL: "/perfTest/identity/build/orchStubUrl"
               IDENTITY_CIMIT_GETCIC: "/perfTest/identity/cimit/getcic"
               IDENTITY_CIMIT_POSTMITIGATION: "/perfTest/identity/cimit/postmitigation"
               IDENTITY_CIMIT_PUTCI: "/perfTest/identity/cimit/putci"
               IDENTITY_CORE_STUB_PASSWORD: "/perfTest/identity/coreStubPassword"
               IDENTITY_CORE_STUB_URL: "/perfTest/identity/coreStubUrl"
               IDENTITY_CORE_STUB_USERNAME: "/perfTest/identity/coreStubUsername"
               IDENTITY_DEV_CORE_URL: "/perfTest/identity/dev/coreUrl"
+              IDENTITY_DEV_ORCH_STUB_URL: "/perfTest/identity/dev/orchStubUrl"
```

Comment:
Note that at the moment each developer's dev environment will have a separate orchestration stub URL (e.g. Mike's is https://orch-dev-mikec.02.core.dev.stubs.account.gov.uk/), will that be a problem?

Reply:
That's a good point - this change was intended for testing the canary environment deployed in dev02 specifically. Happy to name it something more specific in case other devs want to use this as well

Reply:
Added a small doc section on how to change developer's env if needed:
https://govukverify.atlassian.net/wiki/spaces/DID/pages/3179511813/SRE+Dev+Environment+Guide#Performance-tests-in-Dev